release/mk-release-snapshots.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I attribute to these concerns a lower importance because:
- I don't expect portability to be a problem for tens of years at least. Linux is entrenched, and bash is entrenched in Linux
- this is a very constrained automation, with no significant attack surface. If bash had a security issue, there would be much more exposed areas where we have no control

This said, this can be addressed in this case and I see no harm in doing as you suggest